tests/Resources/modules/translationtest/src/Controller/Admin/ExempleController.php

```diff
@@ -24,15 +24,15 @@
  * International Registered Trademark & Property of PrestaShop SA
  */
 
-namespace TranslationTest\Controller\Admin;
+namespace PrestaShop\Module\TranslationTest\Controller\Admin;
 
 use PrestaShopBundle\Controller\Admin\FrameworkBundleAdminController;
 
 class ExempleController extends FrameworkBundleAdminController
 {
     public function indexAction()
     {
-        return $this->render('@Modules/translations/views/templates/admin/index.html.twig', [
+        return $this->render('@Modules/translationtest/views/templates/admin/index.html.twig', [
             'layoutTitle' => $this->trans('Modern controller', 'Modules.Translationtest.Admin'),
```

Comment:
What is it?

Reply:
These, are tests resources, you can ignore everything in tests directory @LouiseBonnard  =)